tests/data/test_checkpoint.py

```diff
@@ -0,0 +1,129 @@
+"""CRUD tests for CheckpointInfo"""
```

Comment:
Rebase from main to remove these changes please

Reply:
which changes specifically? is there something messed up from the rebase? or all good now?